easybuild/tools/environment.py

```diff
@@ -93,6 +94,12 @@ def setvar(key, value):
     _changes[key] = value
     _log.info("Environment variable %s set to %s (%s)", key, value, oldval_info)
 
+    if verbose and build_option('extended_dry_run'):
```

Comment:
the `verbose` indicates whether or not defining this environment variables should be printed under `-x` or not

in some places, defining the environment variables is considered 'internal', i.e. it serves a purposes but is mostly irrelevant so shouldn't 'pollute' the dry run output

example: all the `$EBVAR*` environment variables (which we should get rid of), there are others too